src/cljs/main/broadfcui/page/workspace/monitor/tab.cljs

```diff
@@ -39,7 +40,12 @@
                   (:status submission)])}
       {:header "Method Configuration" :initial-width 300
        :column-data (juxt :methodConfigurationNamespace :methodConfigurationName)
-       :as-text (fn [[namespace name]] (str namespace "/" name))}
+       :as-text (fn [[namespace name]] (str namespace "/" name))
```

Comment:
Potentially too-clever rewrite:
```clojure
:as-text (partial clojure.string/join "/")
```
Which reads, "To convert to text, join with '/'"